immunisations/README.md

```diff
@@ -0,0 +1,149 @@
+# Immunization Mapping
+
+## JSON FHIR > XML HL7
+
+An Immunization is mapped to a ObservationStatement.
+
+| Mapped to (JSON FHIR Immunization field)                 | Mapped from (XML HL7 / other source)                                                                          |
+|----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| id                                                       | `ObservationStatement / id / root]`                                                                           |
+| meta / profile                                           | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Immunization-1"`                 |
+| extension / url                                          | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-VaccinationProcedure-1` |
+| extension / valueExtension / url                         | fixed value = `https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid`                       |
+| extension / valueExtension / valueCodeableConcept / text | `ObservationStatement / code / displayname`                                                                   |
+| extension / url[1]                                       | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-DateRecorded-1`         |
+| extension / valueString                                  | `ObservationStatement / effectiveTime / highValue`                                                            |
+| identifier / system                                      | `ObservationStatement / code / displayname`                                                                   |
+| identifier / value                                       | `ObservationStatement / id`                                                                                   |
+| status                                                   | `ObservationStatement / statusCode`                                                                           |
+| notGiven                                                 | boolean value = `if passed through then set to true, otherwise it's false`                                    |
+| patient / reference                                      | reference to the mapped [patient](../patient/README.md)                                                       |
+| encounter / reference                                    | reference to the associated [encounter] ((../encounter/README.md))                                            |
+| date                                                     | `ObservationStatement / effectiveTime`                                                                        |

```

Comment:
Looks like an additional note gets added to the Immunization if an end date is provided.  Which I guess matches up to what we see in the example below.